### PR TITLE
Add SCNIC python tool

### PR DESCRIPTION
**Pull request by shafferm**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/scnic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: scnic
+  version: {{ version }}
+
+source:
+  url: https://github.com/shafferm/SCNIC/archive/{{ version }}.tar.gz
+  sha256: ddd16c12813b6755440e4fd196e58a1ce7def8fef09f8cb3b3bea046a24962f6
+
+requirements:
+  host:
+    - python >=3
```
Comment (epruesse) [inline recipes/scnic/meta.yaml]: no need to have `>=3`

File: recipes/scnic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - networkx >2
+    - scikit-bio
+    - fastspar
+
+build:
+  skip: True  # [py27]
+
+test:
+  imports:
+    - SCNIC
```
Comment (epruesse) [inline recipes/scnic/meta.yaml]: could you add a 

```
command: 
  - scnic 
```
to try if the script entry point runs?

**Comment by epruesse** [comment]
@bioconda/core - why does this not build anything on osx?

```
09:58:29 BIOCONDA INFO Building and testing subdag 1 of 1 (1 recipes)
09:58:30 BIOCONDA INFO Determining expected packages
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
Adding in variants from internal_defaults
Adding in variants from /Users/distiller/project/miniconda/conda_build_config.yaml
Adding in variants from /Users/distiller/project/miniconda/lib/python3.6/site-packages/bioconda_utils/bioconda_utils-conda_build_config.yaml
Attempting to finalize metadata for scnic
Solving environment: ...working... done
Solving environment: ...working... done
09:59:39 BIOCONDA INFO Nothing to be done for recipe recipes/scnic
09:59:39 BIOCONDA INFO BUILD SUMMARY: successfully built 0 of 1 recipes
```

File: recipes/scnic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/shafferm/SCNIC/archive/{{ version }}.tar.gz
+  sha256: ddd16c12813b6755440e4fd196e58a1ce7def8fef09f8cb3b3bea046a24962f6
+
+
+build:
+  number: 0
+  noarch: True
```
Comment (rvalieris) [inline recipes/scnic/meta.yaml]: the correct syntax is `noarch: python`

Comment (epruesse) [reply recipes/scnic/meta.yaml]: thanks @rvalieris

**Comment by rvalieris** [comment]
@epruesse noarch packages run on any platform, so they don't need to be built on osx.

**Pull request merged**